api/src/test/java/org/pmiops/workbench/db/dao/ParticipantCohortStatusDaoTest.java

```diff
@@ -125,7 +125,7 @@ public void findAllSaveParticipantCohortStatuses() {
     final Object[] sqlParams = {key1.getCohortReviewId()};
     final Integer expectedCount = Integer.valueOf("2");
 
-    assertThat(jdbcTemplate.queryForObject(sql, sqlParams, Integer.class)).isEqualTo(expectedCount);
+    assertThat(jdbcTemplate.queryForObject(sql, Integer.class, sqlParams)).isEqualTo(expectedCount);
```

Comment:
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/jdbc/core/JdbcTemplate.html#queryForObject(java.lang.String,java.lang.Class,java.lang.Object...)

I think the above is what you are providing 
![Screen Shot 2023-05-19 at 9 26 41 AM](https://github.com/all-of-us/workbench/assets/24514630/0d687284-5502-4a67-8186-45dad42cb6c0)
I'm not familiar with what Object ... args would mean? Is that another way of saying array?

Reply:
Yes, I believe that is correct. I think you can either pass as many args as you want one-by-one, or you can pass an array.